src/adafruit_blinka/microcontroller/ft232h/pin.py

```diff
@@ -8,6 +8,9 @@ class Pin:
     OUT = 1
     LOW = 0
     HIGH = 1
+    PULL_NONE = 0
+    PULL_UP = 1
```

Comment:
is this UP and DOWN needed? we dont ever use?

Reply:
Hmm, I thought the definitions are needed for the test whether pull-up/down is supported. I'll check tomorrow Morning.

Reply:
Yes, I think the PULL_UP and PULL_DOWN are needed because module digitalio uses them. So, even if they are not used because pull up/down is not supported by the HW, the are needed for the test of the HW capabilities. 

Please see the digitalio code snippet of class **DigitalInOut** below
```Python
 @pull.setter
    def pull(self, pul):
        if self.direction is Direction.INPUT:
            self.__pull = pul
            if pul is Pull.UP:
                self._pin.init(mode=Pin.IN, pull=Pin.PULL_UP)
            elif pul is Pull.DOWN:
                if hasattr(Pin, "PULL_DOWN"):
                    self._pin.init(mode=Pin.IN, pull=Pin.PULL_DOWN)
                else:
                    raise NotImplementedError(
                        "{} unsupported on {}".format(Pull.DOWN, board_id)
                    )
```

Reply:
ok!

Reply:
Hmm, on long term it might be an idea to refactor the redundant definitions within the Pin classes. However for the moment the definition should fix the problem.